src/index.ts

```diff
@@ -33,14 +33,20 @@ app.get('/status', async (_req, res) => {
   const status = await bot.status();
   jsonExt(status, res);
 });
-app.get('/earnings/:rotationId', async (req, res) => {
-  const rotationId = req.params.rotationId;
-  const data = await bot.storage.earningsFile(Number(rotationId)).get();
+app.get('/bids', async (_req, res) => {
+  const currentFrameId = await bot.currentFrameId();
+  const nextFrameId = currentFrameId + 1;
+  const data = await bot.storage.bidsFile(nextFrameId).get();
   jsonExt(data, res);
 });
-app.get('/biddings/:cohortId', async (req, res) => {
-  const cohortId = req.params.cohortId;
-  const data = await bot.storage.biddingsFile(Number(cohortId)).get();
+app.get('/bids/:frameIdAtCohortActivation', async (req, res) => {
```

Comment:
Is this returning the bids _for_ the frameId that was activated here? I really like some of the consolidation of frameIds in some places, but this is a place that I actually find it very hard to reason through

Reply:
Might be simpler to just call this "frameId" - I would know what that was doing (bids that occurred during that frame).